.gitignore

```diff
@@ -57,4 +102,30 @@ typings/
 # dotenv environment variables file
 .env
 
-*.db
\ No newline at end of file
+
+### Windows ###
+# Windows thumbnail cache files
+Thumbs.db
```

Comment:
dunt think this is necessary

Reply:
Think we should have it in there in case anyone who works on a windows machine commits something to the repo. Usually I setup all my gitignore.io files to include all OS's: Mac, Windows, Linux